app/screens/home/search/search.tsx

```diff
@@ -63,6 +67,15 @@ const getSearchParams = (terms: string, filterValue?: FileFilter) => {
     };
 };
 
+const getPosts = async (serverUrl: string, ids: string[]) => {
```

Comment:
I wanted to leave this comment because I have debated this change while designing this feature. 

I have contemplated whether I should have removed this from `results/index.tsx` and got the values from an observable, but I believe it isn't necessary to observe.  The search results for files and posts won't ever live update, even if a new post with the search term is received via a websocket event. This is because the posts are fetched from the server.  The postIds are then fetched from the WatermellonDB. There is no live updating (as designed)

Reply:
Perhaps, you can move it into a local action